### ISSUE TITLE
:3

### ISSUE THREAD

**Issue by Hacker1254** [Imabot115/ce-EasyAntiCheat-Bypass#1]
:3 https://www.virustotal.com/gui/file/d4672b9488fc3c0565e1835da4fbd0737c87ac4d34dbc481f0cdc3a8ad2e4b56

**Comment by Imabot115** [comment]
Yes hi so the reason it comes up with so many malewares and Trojans is because of Wind64 which is a windows hacking tool all it does is disable Dse aka driver signature enforcement you need this because the ce drivers are not signed so trying to load them will result in a error however disabling dse with Wind64 loads them if you still think this is a virus or something you can run it on a Vm and tell me any results you may find
